test/org/sagebionetworks/bridge/stormpath/StormpathAccountDaoMultipleSubpopulationsTest.java

```diff
@@ -1,109 +0,0 @@
-package org.sagebionetworks.bridge.stormpath;
-
-import static org.junit.Assert.assertEquals;
-
-import java.util.List;
-
-import javax.annotation.Resource;
-
-import org.joda.time.DateTime;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-
-import org.sagebionetworks.bridge.TestUtils;
-import org.sagebionetworks.bridge.models.accounts.Account;
-import org.sagebionetworks.bridge.models.accounts.StudyParticipant;
-import org.sagebionetworks.bridge.models.studies.Study;
-import org.sagebionetworks.bridge.models.subpopulations.ConsentSignature;
-import org.sagebionetworks.bridge.models.subpopulations.Subpopulation;
-import org.sagebionetworks.bridge.services.StudyService;
-import org.sagebionetworks.bridge.services.SubpopulationService;
-
-@ContextConfiguration("classpath:test-context.xml")
-@RunWith(SpringJUnit4ClassRunner.class)
-public class StormpathAccountDaoMultipleSubpopulationsTest {
```

Comment:
Actually, this specific change reverts the fix for ConsentServiceTest. I know for a fact that the version of this test in StormpathAccountDaoTest will break ConsentServiceTest, which is why it was moved to its own file in the first place.